### PR TITLE
Fix markdown formatting in persistence provider Readme

### PR DESCRIPTION
**Pull request by SebastianSchildt**

### CODE REVIEW

File: kuksa-persistence-provider/Readme.md [hunk 1]
```diff
@@ -83,20 +83,27 @@ As the statestore does not make a difference between current and target (actuati
...
   * Install rust nightly which is needed for djson at the moment [11/2024]
-    ```bash
+
+  * ```bash
     rustup toolchain install nightly
     rustup default nightly
     ```
-  * Enable with ```cargo build --features json_djson --no-default-features``````
 
+  * Enable with ```cargo build --features json_djson --no-default-features``````
```
Comment (erikbosch) [inline kuksa-persistence-provider/Readme.md]: Remove the last 3 characters
image

File: kuksa-persistence-provider/Readme.md [hunk 2]
```diff
@@ -129,4 +136,5 @@ cargo build --features json_djson --no-default-features
         # Set Vales in cmd
         setValue Vehicle.Cabin.HVAC.Station.Row4.Passenger.FanSpeed 40
         ```
+
```
Comment (erikbosch) [inline kuksa-persistence-provider/Readme.md:139]: When fixing the comment above also fix `Vales` a few lines above

File: kuksa-persistence-provider/Readme.md (hunk 1 above)
Comment (SebastianSchildt) [reply kuksa-persistence-provider/Readme.md]: Done

File: kuksa-persistence-provider/Readme.md (hunk 2 above)
Comment (SebastianSchildt) [reply kuksa-persistence-provider/Readme.md:139]: Done

**Review by erikbosch — APPROVED** [review]
LGTM

**Pull request merged**